java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -1299,13 +1299,15 @@ public final ColumnVector rollingWindow(RollingAggregation op, WindowOptions opt
 
     long nativePtr = agg.createNativeInstance();
     try {
+      Scalar p = options.getPrecedingScalar();
```

Comment:
:+1: 
I shied from this when #7909 was in development. The `int` values are required for `Table.aggregateWindows()`, while the full `scalar` is for `Table.aggregateRangeWindows()`. At the time, I thought storing these as `scalar` values wasn't necessary. I'm convinced now that the ambiguity is less preferable than the cost of the `scalar`.

Reply:
Yeah I find it a bit odd to force the caller to go through Scalar and build a GPU-side resource but then force it to come back to the CPU in some cases.  But I agree it's nice not to have the ambiguity.